### DISCUSSION TITLE
ability to cancel/delete single contracts instead of whole list

### DISCUSSION THREAD

**Discussion by sashaman** [Royal-Modding/FS19_BetterContracts#17] [Ideas]
![grafik](https://user-images.githubusercontent.com/1388895/116194851-b2cdcd00-a731-11eb-838b-2e07652c468d.png)
how about an additional button to cancel/delete single contracts instead of whole list

**Comment by ghost** [comment]
You can cancel a single contract easily:
Accept, cancel, and dismiss it 😉

**Comment by sashaman** [comment]
right lol 
idk why i had npc relations in mind... 
checked again and didnt find a failed counter or something that has negative impact, seems theres only finishedMissions 
so ignore for now :)
